inc/util/string_utils.hpp

```diff
@@ -46,6 +46,21 @@ namespace ebi
 
         output.swap(ret);
     }
+
+    /**
+     * Temporal implementation for mismatch with 2 starts and 2 ends. It is not in STL for c++11, but it will in c++14.
```

Comment:
For Clang, C++14 is supported from version 3.4, released in Jan 2014 (current 3.8).
For GCC, version 4.9 supports most of the features and 5.x the rest. Support for C++17 is experimental.